scripts/generate_key.py

```diff
@@ -0,0 +1,210 @@
+#!/bin/env python3
+
+import argparse
+import os
+import subprocess
+import json
+
+
+def run_key_generator(key_generator_path, args, capture_stderr=False):
+    result = subprocess.run([key_generator_path] + args, capture_output=True, text=True)
+    if result.returncode != 0:
+        raise Exception(f"Key generator failed: {result.stderr}")
+    return result.stderr if capture_stderr else result.stdout
+
+
+def parse_key_output(output):
+    lines = output.strip().split('\n')
+    return {
+        'secret_phrase': lines[0].split(':')[1].strip(),
+        'ss58_public_key': lines[5].split(':')[1].strip()
+    }
+
+
+def generate_secret_phrase(base_key, name_component, index):
+    if base_key is None:
+        return None
+    return f"{base_key}///{name_component}/{index}"
+
+
+def generate_validator_keys(key_generator_path, num_validators, base_key=None, separate_account=False):
```

Comment:
Minor, non-blocker: Can't we re-use here the `generate_sr25519_keys` method ? Perhaps for better readability we can introduce also a `generate_ed25519_keys` method ?